### PR TITLE
Add-LoginPage-Component.

### LINKED ISSUE #23: [Feature] - LOGIN PAGE
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
Just a clean Login page having several essential entities.

**Describe alternatives you've considered**
N/A

**Additional context**
One can even integrate this with the backend and then it's good to go!

### PR DESCRIPTION
**Pull request by deVedant**
**Linked Issue(s)**
- Resolves #23 

**Describe the changes**
Added a Login Page Component.

**Change(s) location**
tools/components

**Additional context**
N/A

### CODE REVIEW

File: tools/components/Login Page A/component.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    <input type="text" class="block appearance-none w-full bg-white border border-grey-light hover:border-grey px-2 py-2 rounded shadow" placeholder="Your Username">
+                </div>
+
+                <div class="mb-4">
+                    <label class="font-bold text-grey-darker block mb-2">Password</label>
+                    <input type="text" class="block appearance-none w-full bg-white border border-grey-light hover:border-grey px-2 py-2 rounded shadow" placeholder="Your Password">
+                </div>
+
+                <div class="flex items-center justify-between">
+                    <button class="bg-teal-dark hover:bg-teal text-white font-bold py-2 px-4 rounded">
```
Comment (IzMichael) [inline tools/components/Login Page A/component.html]: Why is the text white?

File: tools/components/Login Page A/documentation.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                    <input type="text" class="block appearance-none w-full bg-white border border-grey-light hover:border-grey px-2 py-2 rounded shadow" placeholder="Your Username">
+                </div>
+
+                <div class="mb-4">
+                    <label class="font-bold text-grey-darker block mb-2">Password</label>
+                    <input type="text" class="block appearance-none w-full bg-white border border-grey-light hover:border-grey px-2 py-2 rounded shadow" placeholder="Your Password">
+                </div>
+
+                <div class="flex items-center justify-between">
+                    <button class="bg-teal-dark hover:bg-teal text-white font-bold py-2 px-4 rounded">
```
Comment (IzMichael) [inline tools/components/Login Page A/documentation.md]: White text also here

**Review by IzMichael — CHANGES_REQUESTED** [review]
Can you explain some of these classes for me?

File: tools/components/Login Page A/component.html (hunk 1 above)
Comment (deVedant) [reply tools/components/Login Page A/component.html]: Oh, yes. My bad.
That wouldn't be visible if that's white.
I'll need to change it to Black.

File: tools/components/Login Page A/documentation.md (hunk 2 above)
Comment (deVedant) [reply tools/components/Login Page A/documentation.md]: Even here.

**Review by IzMichael — APPROVED** [review]
Excellent!

**Comment by IzMichael** [comment]
@all-contributors please add @deVedant for Code

**Pull request merged**